### DISCUSSION TITLE
Erasing  simulators between test classes

### DISCUSSION THREAD

**Discussion by alexnot95** [lyndsey-ferguson/fastlane-plugin-test_center#346] [General]
Hi, @lyndsey-ferguson I found that during our test run we use parallel execution through multi_scan with 3 simulators, for example, we have 9 tests in 1 LoginUITest.swift file, we will execute 9 tests on 3 simulators, and when we want to execute another 9 tests in for ex. LogoutUITest.swift file, the simulators will be erased and then 9 tests will start, we can somehow disable erasing simulators between tests and erase simulators only at the end or at the start of the test run? Thank you

**Comment by alexnot95** [comment]
Maybe ```reset_simulator: true``` affect it?

**Reply by lyndsey-ferguson** [reply]
Hello @alexnot95, have you tried setting `reuse_simulators_for_parallel_testruns` to true?

**Reply by lyndsey-ferguson** [reply]
Here are the other options you can set for `multi_scan`:
https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/docs/feature_details/multi_scan.md (that's linked from the main README for the project repo).

**Reply by alexnot95** [reply]
no, it will took already created simulators? And can I use ```reset_simulator: true``` and ``` reuse_simulators_for_parallel_testruns: true``` in one lane?

**Reply by alexnot95** [reply]
![Screenshot 2021-07-08 at 17 49 33](https://user-images.githubusercontent.com/37154209/124943167-f2a80f00-e014-11eb-838f-3dd57381d2ff.png)
is it a duplicate?

**Reply by lyndsey-ferguson** [reply]
Yes, it is indeed. This is autogenerated so it looks like maybe I had a merge conflict or I added some extra stuff. I'll fix it.

**Reply by lyndsey-ferguson** [reply]
`reset_simulator` wipes the simulator, so if want to avoid that, set it to false.

**Reply by lyndsey-ferguson** [reply]
Did this information help you?
